templates/vue/asconfig.js

```diff
@@ -0,0 +1,11 @@
+const compile = require('near-sdk-as/compiler').compile
+
+compile('assembly/main.ts', // input file
+  'out/main.wasm',    // output file
+  [
+    //   "-O1",          // Optional arguments
+    '--debug',
+    '--measure',         // Shows compiler runtime
+  ], {
+    verbose: true     // Output the cli args passed to asc
+  })
```

Comment:
While you were working, `asconfig.js` was deprecated! Please update to `near-sdk-as@1.0.x`. This will let you use `asconfig.json` instead of `asconfig.js`, and you'll need to update some of the `package.json` scripts to use a new `asb` command to build the contract. Use #425 as a guide. 

Also, just to let you know, sometime soon I want to reorganize the assemblyscript contract folder so that we don't need this assemblyscript-specific stuff in the frontend project root directory, and we'll also need fewer exceptions in the `index.js` file (no more `assembly` folder vs a `contract` folder, for example). If that had been in place before you started, then the change from `asconfig.js` to `asconfig.json` would not have required any changes in your PR.

Reply:
ok, I've upgraded to near-sdk-as@1.0.x